compiler/rustc_codegen_llvm/src/debuginfo/namespace.rs

```diff
@@ -29,7 +30,7 @@ pub fn item_namespace(cx: &CodegenCx<'ll, '_>, def_id: DefId) -> &'ll DIScope {
 
     let namespace_name = match def_key.disambiguated_data.data {
         DefPathData::CrateRoot => cx.tcx.crate_name(def_id.krate),
-        data => data.as_symbol(),
+        data => Symbol::intern(&data.to_string()),
```

Comment:
Since this is cached, I think it's better to use `.to_string()` on the `crate_name` (since crates are rarer) than `Symbol::intern` everything else.

Or if you want you can probably make this as cheap as possible like this:

```rust
    let crate_name_as_str;
    let name_to_string;
    let namespace_name = match def_key.disambiguated_data.data {
        DefPathData::CrateRoot => {
            crate_name_as_str = cx.tcx.crate_name(def_id.krate).as_str();
            &crate_name_as_str
        }
        data => {
            name_to_string = data.to_string();
            &name_to_string
        }
    };
```